examples/partial_execution/README.md

```diff
@@ -0,0 +1,34 @@
+# Partial execution
+
```

Comment:
Maybe link to the `gantry.def.yml` and `gantry.env.yml` in a bullet list possible like:

 * gantry.def.yml: Pipeline definition for this example
 * gantry.env.yml: Default pipeline environment for this example.

You could add this section to all the examples.